app/main.py

```diff
@@ -16,4 +16,11 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films,
+                                marks,
+                                collection_of_coins],
+                    "immutable": [lucky_number,
+                                  pi,
+                                  one_is_a_prime_number,
+                                  name,
+                                  profile_info]}
```

Comment:
the solution is correct, but it seems not pretty at all 😢 please fix your code styling because it's not readable for now. for example, you can write dictionary keys in a new line -> row would have more width